libraries/AP_DDS/dds_xrce_profile.xml

```diff
@@ -30,4 +30,29 @@
       </historyQos>
     </topic>
   </data_writer>
+  <topic profile_name="navsatfix0__t">
```

Comment:
Add the 0 since there may be multiple GPS's simultaneously reporting in the future.